cases/arithmetic_decimal/sqrt_decimal.yaml

```diff
@@ -0,0 +1,112 @@
+base_uri: https://github.com/substrait-io/substrait/blob/main/extensions/substrait/extensions/functions_arithmetic_decimal.yaml
+function: sqrt
+cases:
+  - group:
+      id: basic
+      description: Basic examples without any special cases
+    args:
+      - value: 25
+        type: decimal<38,0>
```

Comment:
Is there a reason to have the scale as 38 instead of 2?

Reply:
no particular reason, Since lower precison value could fit in higher precison so used it
Fixed it now to use exact precsion. Thanks

Reply:
If you decide to use exact precision, we should be consistent across all the various function test cases.  I'm also with just sticking to use the higher precision.  Just make sure it's consistent.

Reply:
sure. I am thinking to use exact precision that way it looks consistent with the tests which requires exact precision

I will fix other tests too

Reply:
lmk once all the other tests use the exact precision too if you decide to go that route.

Reply:
I fixed it in this and other PRs. Kept is based on Substrait. Also updated PR description to reflect same